share/wake/lib/system/remote_cache_api.wake

```diff
@@ -631,6 +632,10 @@ export def rscApiGetFileBlob ((CacheSearchBlob _ uri): CacheSearchBlob) (path: S
     Pass path
 
 ## --- Helper functions ---
+
+def contentTypeFromSize size =
+    if size < 95 then Some "blob/small" else None
```

Comment:
Did we determine ~100 was the right limit still?

Reply:
nope, we talked about 256 but I'm kicking that down the road